### PR TITLE
FOGL-2847 syslog file path fixes for RHEL/ CentOS

### PR DESCRIPTION
**Pull request by praveen-garg**
set `/var/log/messages` permissions to 644.

### CODE REVIEW

File: python/foglamp/services/core/api/support.py [hunk 1]
```diff
@@ -18,7 +19,12 @@
 
 _FOGLAMP_DATA = os.getenv("FOGLAMP_DATA", default=None)
 _FOGLAMP_ROOT = os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
+
 _SYSLOG_FILE = '/var/log/syslog'
+
+if ('centos' in platform.platform()) or ('redhat' in platform.platform()):
+    _SYSLOG_FILE = '/var/log/message'
```
Comment (MarkRiddoch) [inline python/foglamp/services/core/api/support.py]: /var/log/messages

File: python/foglamp/services/core/support.py [hunk 2]
```diff
@@ -30,6 +31,9 @@
 _NO_OF_FILES_TO_RETAIN = 3
 _SYSLOG_FILE = '/var/log/syslog'
 
+if ('centos' in platform.platform()) or ('redhat' in platform.platform()):
+    _SYSLOG_FILE = '/var/log/message'
```
Comment (MarkRiddoch) [inline python/foglamp/services/core/support.py]: /var/log/messages

File: python/foglamp/services/core/api/support.py (hunk 1 above)
Comment (praveen-garg) [reply python/foglamp/services/core/api/support.py]: Thanks, fixed!

File: python/foglamp/services/core/support.py (hunk 2 above)
Comment (praveen-garg) [reply python/foglamp/services/core/support.py]: Thanks, fixed!

**Pull request merged**